### PR TITLE
Containers: Enable log collection for autoyast installations

### PR DESCRIPTION
**Pull request by jlausuch**
Minor additions to collect `save_y2logs` logs from autoyast installation in container hdd tests.

### CODE REVIEW

File: tests/autoyast/save_y2logs.pm [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+# Summary: run save_y2logs and upload the generated tar.bz2
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
```
Comment (jlausuch) [inline tests/autoyast/save_y2logs.pm:10]: @OleksandrOrlov I hope you don't mind this :)

**Comment by jlausuch** [comment]
[15-SP4](https://openqa.suse.de/tests/overview?distri=sle&version=15-SP4&build=43.1&groupid=313&test=create_hdd_autoyast_containers)

[All container hdd tests](https://openqa.suse.de/tests/overview?flavor=Container-Host&groupid=377&build=jlausuch%23containers_add_autoyast_logs)

File: data/containers/autoyast_containers.xml.ep [hunk 2]
```diff
@@ -121,7 +121,7 @@
     </packages>
     <patterns config:type="list">
       <pattern>base</pattern>
-      % if ($get_var->('VERSION') =~ /15|15-SP1|15-SP2|15-SP3|15-SP4/ and $get_var->('ARCH') !~ /aarch64/) {
+      % if ($get_var->('VERSION') =~ /15/) {
```
Comment (cfconrad) [inline data/containers/autoyast_containers.xml.ep:124]:
```suggestion
      % if ($get_var->('VERSION') =~ /^15/) {
```
I know it is picky, but I imagine, there is a 12-SP15 ;)

**Review by cfconrad — APPROVED** [review]
NICE!
only the "All container hdd tests" link has all jobs canceled...

File: data/containers/autoyast_containers.xml.ep (hunk 2 above)
Comment (jlausuch) [reply data/containers/autoyast_containers.xml.ep]: done, and link updated

**Review by OleksandrOrlov — APPROVED** [review]

**Pull request merged**